src/660.json

```diff
@@ -8,6 +8,13 @@
                "title" : "Registration for The Perl and Raku Conference 2024 is open!",
                "ts" : "2024.03.15",
                "url" : "https://news.perlfoundation.org/post/registration_perl_and_raku_conference_2024"
+            },
+            {
+               "author" : "",
+               "text" : "Teaming with the french mongers, Open Food Facts organize a Hackathon around the use of Perl. Perl is used for the web server which is the heart of the project. It will take place in Paris, on Saturday March 23 and Sunday March 24. Remote participation is possible.",
```

Comment:
`Teaming with the french mongers, Open Food Facts organize a Hackathon around the use of Perl by it's web server which is the heart of the project. It will take place in Paris, on Saturday March 23 and Sunday March 24. Remote participation is possible.`

Sounds better and shorter.